src/core/dom/getActiveElement.js

```diff
@@ -14,13 +14,15 @@
  * Same as document.activeElement but wraps in a try-catch block. In IE it is
  * not safe to call document.activeElement if there is nothing focused.
  *
- * The activeElement will be null only if the document body is not yet defined.
+ * The activeElement will be null only if the document or document body is not yet defined.
  */
 function getActiveElement() /*?DOMElement*/ {
+  var localDocument = (typeof document === 'undefined') ? {} : document;
```

Comment:
I think if we go with this I would ultimately just shorten it - we know that we're going to return `undefined` when `typeof document === 'undefined'` - no point assigning to an empty object and entering a try/catch.